Compile errors on Ubuntu 20.04

Tried to compile it for Ubuntu 20.04, but got errors:

```
tools/quake3/q3map2/light_ydnar.c: In function ‘FloodLightForSample’:
tools/quake3/q3map2/light_ydnar.c:4112:7: error: ‘lm’ undeclared (first use in this function); did you mean ‘tm’?
 4112 |   if( lm->styles[lightmapNum] != LS_NORMAL && lm->styles[lightmapNum] != LS_NONE ) // isStyleLight
      |       ^~
      |       tm
tools/quake3/q3map2/light_ydnar.c:4112:7: note: each undeclared identifier is reported only once for each function it appears in
tools/quake3/q3map2/light_ydnar.c:4112:18: error: ‘lightmapNum’ undeclared (first use in this function); did you mean ‘lightmapGamma’?
 4112 |   if( lm->styles[lightmapNum] != LS_NORMAL && lm->styles[lightmapNum] != LS_NONE ) // isStyleLight
      |                  ^~~~~~~~~~~
      |                  lightmapGamma
scons: *** [build/release/q3map2/tools/quake3/q3map2/light_ydnar.o] Error 1
scons: building terminated because of errors.
```

--- a/tools/quake3/q3map2/light_ydnar.c
+++ b/tools/quake3/q3map2/light_ydnar.c
@@ -4109,9 +4109,6 @@ float FloodLightForSample( trace_t *trace ){
 		/* set endpoint */
 		VectorMA( trace->origin, dd, direction, trace->end );
 
-		if( lm->styles[lightmapNum] != LS_NORMAL && lm->styles[lightmapNum] != LS_NONE ) // isStyleLight
-			continue;
-
 		//VectorMA( trace->origin, 1, direction, trace->origin );
 
 		SetupTrace( trace );

